AdminUi/src/AdminUi/ClientApp/src/app/services/logger-server-service/logger-server.service.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpBackend, HttpRequest } from "@angular/common/http";
+import { Injectable, NgZone } from "@angular/core";
+import { INGXLoggerMetadata, NGXLoggerServerService, NgxLoggerLevel } from "ngx-logger";
+import { AuthService } from "../auth-service/auth.service";
+import { Observable } from "rxjs";
+
+@Injectable({
+    providedIn: "root"
+})
+export class LoggerServerService extends NGXLoggerServerService {
+    isLoggedIn$: Observable<boolean> | undefined;
+
+    constructor(httpBackend: HttpBackend, ngZone: NgZone, private authService: AuthService) {
+        super(httpBackend, ngZone);
+    }
+
+    protected override alterHttpRequest(httpRequest: HttpRequest<any>): HttpRequest<any> {
+        this.isLoggedIn$ = this.authService.isLoggedIn;
```

Comment:
What is the dollar sign at the end of the variable for?

Reply:
1. Why don't you use the HttpClient then?
2. I still don't understand what the dollar sign is for.